package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "exceljs",
-  "version": "1.12.1",
+  "version": "1.12.2",
```

Comment:
Please don't edit this - it is managed during the release/publish process

Reply:
ok